### PR TITLE
python39packages.mpldatacursor: init at 0.7.1

### PR DESCRIPTION
**Pull request by bzizou**
###### Motivation for this change

Prerequisite for HP2P (https://github.com/NixOS/nixpkgs/pull/96998)

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/mpldatacursor/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  # No tests included in archive
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/joferkington/mpldatacursor";
+    description = "Interactive data cursors for matplotlib";
+    license = licenses.mit;
+  };
+
+}
```
Comment (fabaff) [inline pkgs/development/python-modules/mpldatacursor/default.nix:27]:
```suggestion
  };
}
```

**Review by fabaff — COMMENTED** [review]
Could you please update the commit messages? Should looks like `python3Packages: mpldatacursor: init at 0.7.1`. Thanks

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 135534` at 7e6f2ba1 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.mpldatacursor</li>
    <li>python39Packages.mpldatacursor</li>
  </ul>
</details>

File: pkgs/development/python-modules/mpldatacursor/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+buildPythonPackage rec {
+  pname = "mpldatacursor";
+  version = "0.7.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "738afeb4c71e1586796b30eddedfc4790619062f745874cdf1e45336a32df0bf";
+  };
+
+  propagatedBuildInputs = [ matplotlib  ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/mpldatacursor/default.nix:16]:
```suggestion
  propagatedBuildInputs = [ matplotlib ];
```

File: pkgs/development/python-modules/mpldatacursor/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  # No tests included in archive
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/mpldatacursor/default.nix:19]: Can we fetch from github instead?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Also please name the commit ``python39Packages.mpldatacursor: init at 0.7.1``.

File: pkgs/development/python-modules/mpldatacursor/default.nix (hunk 3 above)
Comment (bzizou) [reply pkgs/development/python-modules/mpldatacursor/default.nix:21]: Sure!

**Comment by markuskowa** [comment]
@GrahamcOfBorg build python3Packages.mpldatacursor

**Review by markuskowa — APPROVED** [review]

**Pull request merged**